net.xmind.ZEN.yml

```diff
@@ -0,0 +1,67 @@
+id: net.xmind.ZEN
+base: io.atom.electron.BaseApp
+base-version: '18.08'
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: XMind
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --env=GTK_IM_MODULE=xim
+cleanup:
+  - "*.a"
+  - "*.la"
+  - /include
+  - /lib/pkgconfig
+  - /lib/cmake
+modules:
+  - name: xmind
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 apply_extra.sh /app/bin/apply_extra
+      - install -Dm644 ${FLATPAK_ID}.appdata.xml -t /app/share/appdata/
+      - install -Dm755 /usr/bin/desktop-file-edit -t /app/bin/
+      - install -Dm755 /usr/bin/ar -t /app/bin/
+      - cp -a /usr/lib/x86_64-linux-gnu/libbfd* /app/lib
+      - ln -s ../extra/XMind\ ZEN/XMind /app/bin/XMind
+    sources:
+      - type: script
+        dest-filename: apply_extra.sh
+        commands:
+          - ar x xmind.deb data.tar.xz
+          - tar fx data.tar.xz
+          - rm data.tar.xz xmind.deb
+          - mv opt/* .
+          - rmdir opt
+          - mv usr export
+          - rm -fr export/share/doc
+          - desktop-file-edit --set-key="Exec" --set-value="XMind %U" export/share/applications/XMind.desktop
+          - desktop-file-edit --set-key="Icon" --set-value="net.xmind.ZEN" export/share/applications/XMind.desktop
+          - desktop-file-edit --set-key="X-Flatpak-RenamedFrom" --set-value="XMind.desktop;" export/share/applications/XMind.desktop
+          - rename "XMind." "net.xmind.ZEN." export/share/{icons/hicolor/*/*,applications}/XMind.*
+
+      - type: file
+        path: net.xmind.ZEN.appdata.xml
+
+      - type: extra-data
+        filename: xmind.deb
+        only-arches:
+          - 'x86_64'
+        url: https://dl2.xmind.cn/XMind-ZEN-for-Linux-64bit.deb
```

Comment:
Does the vendor also provide versioned tarball? It may be problematic to keep it possible install as we still lack automation for that.

Reply:
Not quite possible for me to change, though I have sent a mail to the vendor.